src/sentry/integrations/slack/endpoints/command.py

```diff
@@ -1,20 +1,27 @@
 import logging
 from typing import Mapping, Sequence, Tuple
 
-from django.http import Http404, HttpResponse
+from django.http import HttpResponse
+from rest_framework import status
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.base import Endpoint
+from sentry.integrations.slack.endpoints.action import LINK_IDENTITY_MESSAGE
+from sentry.integrations.slack.message_builder.disconnected import SlackDisconnectedMessageBuilder
 from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
 from sentry.integrations.slack.requests.base import SlackRequestError
 from sentry.integrations.slack.requests.command import SlackCommandRequest
-from sentry.integrations.slack.views.link_team import build_linking_url
-from sentry.models import Identity, IdentityProvider
+from sentry.integrations.slack.views.link_identity import build_linking_url
+from sentry.integrations.slack.views.link_team import build_team_linking_url
+from sentry.integrations.slack.views.unlink_identity import build_unlinking_url
 
 logger = logging.getLogger("sentry.integrations.slack")
 LINK_TEAM_MESSAGE = "Link your Sentry team to this Slack channel! <{associate_url}|Link your team now> to receive notifications of issues in Sentry in Slack."
 LINK_USER_MESSAGE = "You must first link your identity to Sentry by typing /sentry link. Be aware that you must be an admin or higher in your Sentry organization to link your team."
+UNLINK_USER_MESSAGE = "<{associate_url}|Click here to unlink your identity.>"
+NOT_LINKED_MESSAGE = "You do not have a linked identity to unlink."
+ALREADY_LINKED_MESSAGE = "You are linked as `{username}`."
```

Comment:
hmm okay I'm running through this locally and maybe this could be more clear if it said "You are **already** linked as {username}`

Reply:
You're right. I was using `/sentry link` as a `whoami` but maybe that should be its own command?